scripts/install.sh

```diff
@@ -130,7 +130,7 @@ install_tool() {
                 $PACKAGER tap isen-ng/dotnet-sdk-versions
                 install_with_sudo dotnet-sdk3-1-400 --cask
             else 
-                install_with_sudo aspnetcore-runtime-3.1
+                install_with_sudo dotnetcore-3.1-aspnetruntime
```

Comment:
We need add a condition for msys which is git bash or else wsl or Linux will break. Thank you for catching this.

Reply:
Yeah, sounds like this should handle it?

Also, pkm choco reference - https://community.chocolatey.org/packages/dotnetcore-3.1-aspnetruntime 

Please do recheck if previous was ever working. Or is this the case that `aspnetcore-runtime-3.1` works for the `ubuntu` and not for the windows?

```*.sh
install_with_sudo() {
    if [[ $OSTYPE == "linux"* ]]; then
        sudo $PACKAGER install $1 -y
    else
        $PACKAGER install $2 $1
    fi
}
```